### PR TITLE
Lock in static node version.

### PR DESCRIPTION
**Pull request by TheBox193**
Testing framework fails on later versions of node and should be updated.
This will lock node version at the latest stable point until updates to testing are done.

### CODE REVIEW

**Comment by TheBox193** [comment]
I believe tests are failing do the the node version Travis is using.

It seems I'm unable to lock it in though.

**Comment by gcochard** [comment]
Thanks fo this!

You'll want to edit the `travis.yml` file rather than `.nvmrc`. 

https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Even-more-Nodejs-versions

**Comment by gcochard** [comment]
I just looked at one commit and not the pr as a whole. You did the right thing but I believe Travis doesn't test changes to the Travis settings in PRs for security reasons. We will need to merge before it passes. I will verify and merge if confirmed.

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ sudo: false
 language: node_js
 node_js:
   - 'node'
```
Comment (gcochard) [inline .travis.yml]: Try removing this line as well

**Pull request closed without merging**